target_csv/target.py

```diff
@@ -11,12 +11,27 @@ class TargetCSV(Target):
 
     name = "target-csv"
     config_jsonschema = th.PropertiesList(
+        th.Property(
+            "output_path",
+            th.StringType,
+            description=(
+                "Filesystem path where to store output files. "
+                "By default, the current working directory will be used."
+            ),
+        ),
+        th.Property(
+            "destination_path",
+            th.StringType,
+            description=(
+                "Filesystem path where to store output files. "
+                "Compatibility property to `hotgluexyz` variant."
```

Comment:
If this property is an alias of `output_path`, can we add that to the description?

Reply:
Hi Edgar. Thanks for the review. Sure, I've adjusted the patch to include your suggestions, and also rebased on `main`.